sdks/models/src/main/avro/json/occurrence-json-record.avsc

```diff
@@ -110,7 +121,19 @@
       {"name":"mediaTypes","type":{"type":"array","items":"string"},"default":null,"doc":""},
       {"name":"mediaLicenses","type":{"type":"array","items":"string"},"default":null,"doc":""},
       {"name":"multimediaItems","type":{"type":"array","items":"Multimedia"},"default":null,"doc":""},
-      {"name":"verbatim","type":["null","VerbatimRecord"],"default":null,"doc":""}
+      {"name":"verbatim","type":["null","VerbatimRecord"],"default":null,"doc":""},
+      // De-normalised event info
```

Comment:
What is the reason to have denormalized events and measurements ?

Reply:
Denormalised events on occurrences - this aids various breakdowns in use in the event search UI component

<img width="1556" alt="Screen Shot 2022-10-13 at 11 21 29 am" src="https://user-images.githubusercontent.com/444897/195572349-2dcc4ae5-cdf5-49ff-aa84-e25a8d269111.png">

<img width="1557" alt="Screen Shot 2022-10-13 at 11 22 40 am" src="https://user-images.githubusercontent.com/444897/195572341-0adcbf69-2695-4654-b145-e75a760bab74.png">